File: spring-ai-a2a-examples/java/v1/seller-agent/src/main/java/io/github/timo/a2a/spring/ai/agent/example/SellerAgentApplication.java
package io.github.timo.a2a.spring.ai.agent.example;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class SellerAgentApplication {

    public static void main(String[] args) {
        SpringApplication.run(SellerAgentApplication.class, args);
    }
}

File: spring-ai-a2a-examples/v1/seller-agent/src/main/kotlin/io/github/timo/a2a/spring/ai/agent/example/SellerAgentApplication.kt
package io.github.timo.a2a.spring.ai.agent.example

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class SellerAgentApplication

fun main(args: Array<String>) {
    runApplication<SellerAgentApplication>(*args)
}
